Source/Core/VideoCommon/ShaderCache.h

```diff
@@ -211,16 +211,16 @@ class ShaderCache final
     struct Shader
     {
       std::unique_ptr<AbstractShader> shader;
-      bool pending;
+      bool pending = false;
     };
     std::map<Uid, Shader> shader_map;
     LinearDiskCache<Uid, u8> disk_cache;
   };
-  ShaderModuleCache<VertexShaderUid> m_vs_cache;
-  ShaderModuleCache<GeometryShaderUid> m_gs_cache;
-  ShaderModuleCache<PixelShaderUid> m_ps_cache;
-  ShaderModuleCache<UberShader::VertexShaderUid> m_uber_vs_cache;
-  ShaderModuleCache<UberShader::PixelShaderUid> m_uber_ps_cache;
+  ShaderModuleCache<VertexShaderUid> m_vs_cache{};
```

Comment:
If you move `m_api_type` value to the header.  I think you can then remove or default `ShaderCache()`.  Then the `{}` for all of these 'complex types' can probably go away?

Reply:
They aren't needed even without it; I'll remove it.  I think I added them before the `bool pending = false;` in `ShaderModuleCache::Shader`.  (Note that it takes quite some time for the analysis to run, and the analysis doesn't always include all of the warnings if there's more than one for a given file, so I may be initializing things that didn't need to be explicitly initialized).